### PR TITLE
Add new FAQs

### PR DESCRIPTION
**Pull request by sherenem**
This PR updates the FAQs page within the documentation and adds a new page to document how to evaluate a model using a local LLM deployment as the AI judge instead of ChatGPT.

### CODE REVIEW

File: documentation/docs/concepts.md [hunk 1]
```diff
@@ -56,3 +56,14 @@ Quantization is a technique used to reduce the size and memory requirements of a
...
+
+### Perplexity
+
+Perplexity (PPL) is a commonly used evaluation metric. It measures the confidence a model has in its predictions, or in simpler words how 'perplexed' or surprised it is by seeing new data. Lower perplexity means the model is highly confident in the sequence of words it is responding with. The underlying assumption is that the model's confidence is correlated to accuracy. Perplexity actually does not reference the ground truth at all and is usually only used to evaluate how well the model has learned the training data.
```
Comment (pascal-pfeiffer) [inline documentation/docs/concepts.md:67]: > Perplexity actually does not reference the ground truth at all and is usually only used to evaluate how well the model has learned the training data.

What is ground truth here? Perplexity is calculated on the next token predictions, so it always needs some kind of truth.

Comment (sherenem) [reply documentation/docs/concepts.md:67]: Shall I remove this sentence from the answer? 
I took this answer from @meganjkurka's [list of FAQs](https://docs.google.com/document/d/1DM3H62Qq99HJ0vGpSAYS5rVv71HMwXkXUEC4yRkAXL4/edit#heading=h.81cu4n9s1oxi) from Citi.

Comment (pascal-pfeiffer) [inline documentation/docs/concepts.md:67]:
```suggestion
Perplexity (PPL) is a commonly used evaluation metric. It measures the confidence a model has in its predictions, or in simpler words how 'perplexed' or surprised it is by seeing new data. Perplexity is defined as the exponentiated cross-entropy of a sequence of tokens. Lower perplexity means the model is highly confident and accurate in the sequence of tokens it is responding with.
```

**Review by pascal-pfeiffer — APPROVED** [review]
Thank you, lgtm!

**Pull request merged**